keras_nlp/layers/preprocessing/preprocessing_layer.py

```diff
@@ -29,7 +29,10 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._convert_input_args = False
         self._allow_non_tensor_positional_args = True
-        self.built = True
+        # self.built = True
```

Comment:
This should be covered on the new version of the PR. We only set `built` if there is no build method.

Reply:
Got it, fixed.